### PR TITLE
KT-19126 Add replace property initializer with getter in interfaces

### PR DESCRIPTION
**Pull request by neworld**
https://youtrack.jetbrains.com/issue/KT-19126

### CODE REVIEW

File: idea/src/org/jetbrains/kotlin/idea/quickfix/ReplaceInitializerWithGetterFix.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import com.intellij.psi.PsiNameIdentifierOwner
+import org.jetbrains.kotlin.diagnostics.Diagnostic
+import org.jetbrains.kotlin.idea.core.deleteElementAndCleanParent
+import org.jetbrains.kotlin.psi.KtDeclaration
+import org.jetbrains.kotlin.psi.KtFile
+import org.jetbrains.kotlin.psi.KtProperty
+import org.jetbrains.kotlin.psi.KtPsiFactory
+import org.jetbrains.kotlin.psi.psiUtil.getNonStrictParentOfType
+
+/**
```
Comment (yole) [inline idea/src/org/jetbrains/kotlin/idea/quickfix/ReplaceInitializerWithGetterFix.kt]: Please don't add `@author` tags to the code

File: idea/src/org/jetbrains/kotlin/idea/quickfix/ReplaceInitializerWithGetterFix.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * @author Andrius Semionovas
+ * @since 2017-07-31
+ */
+class ReplaceInitializerWithGetterFix(element: KtProperty): KotlinQuickFixAction<KtProperty>(element) {
```
Comment (yole) [inline idea/src/org/jetbrains/kotlin/idea/quickfix/ReplaceInitializerWithGetterFix.kt]: We already have `ConvertPropertyInitializerToGetterIntention` which is offered as an intention action in this context; you should be able to reuse it directly.

File: idea/testData/quickfix/replaceInitializerWithGetter/simple.kt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+// "Replace initializer with getter" "true"
+
+fun String.foo() = "bar"
+
+interface A {
+    val name = <caret>"The quick brown fox jumps over the lazy dog".foo()
```
Comment (yole) [inline idea/testData/quickfix/replaceInitializerWithGetter/simple.kt]: Neither the existing intention action nor your suggested quickfix handle `var` properties correctly. They should create a stub for the setter with no implementation.

**Comment by neworld** [comment]
Updated

**Comment by yole** [comment]
Thanks a lot for your contribution!

**Pull request merged**